database/sql/src/main/scala/cromwell/database/sql/MetadataSqlDatabase.scala

```diff
@@ -78,10 +78,11 @@ trait MetadataSqlDatabase extends SqlDatabase {
                           rootWorkflowIdKey: String,
                           labelMetadataKey: String,
                           limit: Int,
+                          nextSummaryPointerUpdateCeiling: Option[Long],
                           buildUpdatedSummary:
                           (Option[WorkflowMetadataSummaryEntry], Seq[MetadataEntry])
                             => WorkflowMetadataSummaryEntry)
-                         (implicit ec: ExecutionContext): Future[(Long, Long)]
+                         (implicit ec: ExecutionContext): Future[(Long, Long, Long)]
```

Comment:
TOL I guess but that truple really ought to be a type

Reply:
I _think_ @kshakir scared me off making things in `cromwell/database/` into types... but maybe I over-corrected?

Reply:
Yeah, you may be right. We can always come back to this later if that's not the case.